api-server/api-server-common/src/storage/impls/postgres/queries.rs

```diff
@@ -203,6 +277,16 @@ impl<'a, 'b> QueryFromConnection<'a, 'b> {
         )
         .await?;
 
+        self.just_execute(
+            "CREATE TABLE ml_account_balance (
```

Comment:
In other places table name is `ml_address_balance`, I changed this table name to `ml_address_balance`